habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -209,6 +220,19 @@ def __init__(self, config: Config) -> None:
             len(self.sim_config.agents[0].action_space)
         )
         self._prev_sim_obs: Optional[Observations] = None
+        for agent_id, _ in enumerate(self.habitat_config.AGENTS):
```

Comment:
@jturner65 I am pretty sure it needs to be called during reconfigure as well, but when are the objects deleted from the scene? I think we should merge the concept of object / scene state management from Sokoban to the basic sim class @dexter1691